en-US/Conductors_Companion/03_RPG_Preparation_Process/Session_Zero/Characters_Connecting.md

```diff
@@ -1,10 +1,44 @@
 
-[Character Connections](./Characters_Character_Connections.md)
+For individual connections, each player can roll on the _Character Connection_ column to establish a relationship with the character of the player on their right.
+Going once around the game table this way means that every character will have two relationships-one with the player on their right and one with the player on their left.
 
-[Relationships](./Characters_Relationships.md)
+For single-session games, consider establishing a single group relationship for all the characters before the game begins.
+This relationship can directly tie into the story of the adventure, and will speed up the game by eliminating lengthy discussions about how the characters got together.
+
+___
+<!-- markdownlint-disable-next-line no-emphasis-as-heading -->
+_Connections_
+
+| d20 | Group Connection              | Character Connection     | Relationship         |
+|----:|:------------------------------|:-------------------------|:---------------------|
+|  1  | Mercenary company             | Sibling of...            | Adopted siblings     |
+|  2  | Self-employed investigators   | Saved by...              | Mentor and student   |
+|  3  | Official investigators        | Served with...           | Friendly rivals      |
+|  4  | Royal advisors                | Protected by...          | Sage and scribe      |
+|  5  | Thieves' guild                | Adventured with...       | Priest and acolyte   |
+|  6  | Secret society                | Friendly rival of...     | Fellow veterans      |
+|  7  | Religious investigators       | Childhood friend of...   | Ward and guardian    |
+|  8  | Adventuring company           | Magically bound to...    | Spouses              |
+|  9  | Business investigators        | Survived with...         | Buddy cops           |
+| 10  | Assassins' guild              | Escaped with...          | Childhood friends    |
+| 11  | Wizarding school              | Apprentice of...         | Noble and bodyguard  |
+| 12  | Monastic students             | Acolyte of...            | Soul bound           |
+| 13  | Gladiator school              | Idolizes...              | Former prisoners     |
+| 14  | Military specialists          | Drinking buddies with... | Former criminals     |
+| 15  | Spy network                   | Business associate of... | Hunted quarry        |
+| 16  | Constabulary                  | Lost a bet to...         | Pact bound           |
+| 17  | Magically bound servants      | Indebted to...           | Apocalypse survivors |
+| 18  | Divinely inspired             | Trained by...            | Savior and saved     |
+| 19  | Protectors of the common folk | Dueling partner of...    | Business partners    |
+| 20  | Seekers of vengeance          | On the run with...       | Master and servant   |
```

Comment:
I made this one large table to save space.
<img width="502" height="507" alt="The rendered table" src="https://github.com/user-attachments/assets/5ef311a3-531f-4830-af5b-d001462dca9c" />